### DISCUSSION TITLE
Incorrect axis of rotation for ABB IRBP A-250

### DISCUSSION THREAD

**Discussion by AlexHva** [visose/Robots#201] [Q&A] [answered]
Hello, 
I added the positioner to a robot in grasshopper. I ran into the problems trying to define the joint's axis for rotation. I have modified the parameters in the xml file based on all the values from the positioner (also the 'a' and 'd') joints, but it does not seem to be setup in the correct way. 

For reference I have included foto's of the xml and 3dm files and a changed version of the positioner test files i tried.

![robot problems2](https://user-images.githubusercontent.com/119063157/204014816-bde90396-54d6-4824-ba6b-53783726e5bd.P
![robot problems](https://user-images.githubusercontent.com/119063157/204014824-a6fbbab4-dd97-44a4-aee0-a2a48adf1fb3.PNG)
NG)
onedrive link for the files
https://icthva-my.sharepoint.com/:u:/g/personal/a_jongejans_hva_nl/ESakyR0gq8NDueSkiD3-KwsBB1W6Qsw1DUp8XxK_U4b_pg?e=dxdBki

It's been quite some time I have been busy troubleshooting this, therefore, I would really appreciate your help on this. Thank you in advance! 

Looking forward to your response!

**Comment by visose** [comment, marked as answer]
Just rotate the positioner model 90 degrees around Z.
